content/guides/references/migration-guide.md

```diff
@@ -38,7 +38,9 @@ Related notes:
 
 ```js
 {
-  baseUrl: 'http://localhost:1234'
+  e2e: {
```

Comment:
Revert this change, it will be addressed in https://github.com/cypress-io/cypress-documentation/pull/4365